docs/connectors.md

```diff
@@ -0,0 +1,48 @@
+# Conduit Connectors
+
+Connectors are an integral part of Conduit. Conduit ships with a couple of connectors that are built into the service to help developers bootstrap pipelines much more quickly. The built-in connectors include Postgres, File, Random Data Generator, Kafka and Amazon S3 connectors.
+
+The difference between Conduit connectors and those you might find from other services is that Conduit connectors are Change Data Capture-first (CDC). CDC allows your pipeline to only get the changes that have happened over time instead of pulling down an entire upstream data store and then tracking diffs between some period of time. This is critical for building real-time event-driven pipelines and applications. But, we'll note where connectors don't have CDC capabilities.
```

Comment:
> But, we'll note where connectors don't have CDC capabilities.

In the connector repos, not here, right?

Reply:
For now, in the connector repos.